app/services/support_interface/send_duplicate_match_email.rb

```diff
@@ -7,7 +7,11 @@ def initialize(candidate)
     end
 
     def call
-      CandidateMailer.fraud_match_email(@candidate.current_application).deliver_later
+      CandidateMailer.fraud_match_email(@candidate.current_application, submitted).deliver_later
+    end
+
+    def submitted
+      @candidate.application_forms(&:application_choices).any?(&:submitted?)
```

Comment:
I don't think this is correct because they could have a previous application form (in the previous cycle, or an A1 application) that isn't relevant here